msisdn-gateway/e2e-test/test/control-script.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import pexpect
+import time
+import sys
+import re
+import argparse
+import json
+from twilio.rest import TwilioRestClient
+
+host = "https://msisdn.services.mozilla.com"
+host_number = "+12182967993"
```

Comment:
I assume this is a legit number. Are we cool with adding it to source control here, or do we want to do some env variable thing here?